src/components/Attribution/Attribution.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
This is neat! I'm glad to see we're thinking about this sort of thing up front instead of having to shoehorn it in last minute when we start going public and realize a license calls for it.

Reply:
Yeah, I started doing this in another one of my projects. I really like [flaticon](https://www.flaticon.com/) and most of their free icons require attribution. I'm not actually rendering this component anywhere as of yet, but it should be easy to throw it in wherever later.